app/controllers/v1/profiles_controller.rb

```diff
@@ -12,6 +12,11 @@ def show
       render_json profile
     end
 
+    def destroy
+      authorize profile
+      render_json profile.destroy, status: :accepted
```

Comment:
Just to make sure I understand - this is `accepted` because of the ProfileHost relationship, is that right? From what I see, this returns the deleted profile in the body just right

Reply:
This is part of the JSON spec: https://jsonapi.org/format/#crud-deleting-responses

> If a deletion request has been accepted for processing, but the processing has not been completed by the time the server responds, the server MUST return a 202 Accepted status code.

Basically, because we're queuing a job for any part of the deletion, it must return a 202.